roles/openshift_facts/library/openshift_facts.py

```diff
@@ -915,7 +916,9 @@ def set_version_facts_if_unset(facts):
         facts['common']['version_gte_3_5_or_1_5'] = version_gte_3_5_or_1_5
         facts['common']['version_gte_3_6_or_1_6'] = version_gte_3_6_or_1_6
 
-        if version_gte_3_4_or_1_4:
```

Comment:
I've updated the version defaults as you mentioned.  Where would I go to find what needed to be done for the new default predicates/priorities for the 3.5/1.5 release?